circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Although [this](https://github.com/SwiftGen/StencilSwiftKit/blob/master/rakelib/utils.rake#L71) is preventing the tests to run on [CircleCI](https://circleci.com/gh/SwiftGen/StencilSwiftKit/120).
Have any suggestion?